tools/CustomTasks/BatchMergeXaml.cs

```diff
@@ -36,6 +36,9 @@ public class BatchMergeXaml : Task
         [Required]
         public ITaskItem[] N19H1Pages { get; set; }
 
+        [Required]
+        public ITaskItem[] T21H1Pages { get; set; }
```

Comment:
Because of this file's additions, we will need to publish a new version of CustomTasks.dll. I am waiting for this PR to get reviewed & approved before I take care of that publication. I don't want to push a new CustomTasks.dll prematurely.